Chiazokam/floof
# Implement Search

#### What does this PR do?
This PR implements the search feature on the Get all banks API

---

#### Description of Task to be completed?
- implement search on API
- set up eslint

---

#### How should this be manually tested?

- Clone this repo
- cd into `bank-search-03`
- run npm run start:dev
- check the docs for API samples

---

#### Any background context you want to provide?

N/A

---



#### Screenshots (if appropriate)

N/A

---

#### Questions:

N/A

---
